pages/tutorial.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="flex flex-column justify-between items-center tc vh-100">
+    <h1>{{ title }}</h1>
+    <SpectrumGame :turns="tutorialTurns" :next-path="nextPath" />
```

Comment:
I replaced this file with [/pages/ch1/play.vue](https://github.com/phivk/tolerant-futures/blob/a97582729e2ee825834f379ad415e450d23510b9/pages/ch1/play.vue) 
Let's not reintroduce the old page structure.